src/www/interfaces.php

```diff
@@ -140,7 +140,10 @@ function parse_xml_regdomain(&$rdattributes, $rdfile = '', $rootobj = 'regulator
     $parsed_xml = array();
 
     if (file_exists('/tmp/regdomain.cache')) {
-        $parsed_xml = unserialize(file_get_contents('/tmp/regdomain.cache'));
+        $parsed_xml = @unserialize(@file_get_contents('/tmp/regdomain.cache'), ['allowed_classes' => false]);
+        if (!is_array($parsed_xml)) {
+            $parsed_xml = array();
+        }
```

Comment:
```suggestion
        $parsed_xml = unserialize(file_get_contents('/tmp/regdomain.cache'), ['allowed_classes' => false]) ?? [];
```

if it's not an array beforehand it should still error later IMO